app/models/Level.coffee

```diff
@@ -39,6 +39,8 @@ module.exports = class Level extends CocoModel
     tmap = {}
     tmap[t.thangType] = true for t in o.thangs ? []
     sessionHeroes = [session?.get('heroConfig')?.thangType, otherSession?.get('heroConfig')?.thangType]
+    if @isType('ladder')
+      sessionHeroes = []  # Don't load session heroes in straight ladder matches
```

Comment:
Not sure I follow why we have this edge case for `ladder`. May not have the context yet.

Reply:
I think players' LevelSessions may get automatically initialized to have some `heroConfig` in there. Maybe not, if [this](https://github.com/codecombat/codecombat/blob/aa6340c297a8907ee4e778f1dd512817948b1bc9/app/lib/LevelLoader.coffee#L275) doesn't get run and it's nowhere else? But, if it does somehow have a `heroConfig`, I want to make sure we don't load the stuff for the hero Thangs, because I explicitly want the `heroConfig` to not matter at all for this level type.